README.md

```diff
@@ -28,13 +28,7 @@
 ### React Native >= 0.60
 
 ```sh
-npx react-native init MyApp --template react-native-template-typescript@next
-```
-
-### React Native <= 0.59
```

Comment:
We could leave an old version in Readme for 0.59 (the one that was published as `latest` right now).

Reply:
Ok there's no point to do that probably, everyone can use the new CLI.

Reply:
The thing is, in the old CLI, there's no support for using tagged templates (https://github.com/react-native-community/cli/issues/475). So to use the old template, we'd have to release a new package with a new name otherwise it won't work. Therefore I'd rather drop support for the old CLI.